app/uk/gov/hmrc/tariffclassificationfrontend/views/ReferralReport.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.tariffclassificationfrontend.views
+
+import uk.gov.hmrc.tariffclassificationfrontend.models.{Queue, ReportResult}
+
+class ReferralReport(results: Seq[ReportResult]) {
+
+  lazy val count: Int = results.map(_.size).sum
+
+  lazy val average: Int = Math.round(results.flatMap(_.value).sum.toDouble / count).toInt
```

Comment:
Again, can count be zero?

Reply:
Count will never be zero :) there wouldnt be a ReportResult for it if it was

Reply:
I thought that'd be the case, but worth asking the question.